about/about-kubernetes-ingress.md

```diff
@@ -0,0 +1,130 @@
+---
+title: About Kubernetes ingress
+description: Learn about Kubernetes ingress!
+---
+
+> <span class="glyphicon glyphicon-info-sign"></span> This guide provides optional background education, including
+> education that is not specific to {{site.prodname}}.
+{: .alert .alert-info}
+
+In this guide you will learn:
+- What is Kubernetes Ingress?
+- Why use ingress?
+- What are the differences between different ingress implementations?
+- How does ingress and network policy interact?
+- How does ingress and services fit together under the covers?
+
+### What is Kubernetes Ingress?
+
+Kubernetes Ingress builds on top of Kubernetes [Services]({{site.baseurl}}/about/about-kubernetes-services) to provide
+load balancing at the application layer, mapping HTTP and HTTPS requests with particular domains or URLs to Kubernetes
+services. Ingress can also be used to terminate SSL / TLS before load balancing to the service.
+
+The details of how Ingress is implemented depend on which {% include open-new-window.html text='Ingress Controller'
+url='https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/' %} you are using. The Ingress
+Controller is responsible for monitoring Kubernetes {% include open-new-window.html text='Ingress'
+url='https://kubernetes.io/docs/concepts/services-networking/ingress/' %} resources and provisioning / configuring one
+or more ingress load balancers to implement the desired load balancing behavior.
+
+Unlike Kubernetes services, which are handled at the network layer (L3-4), ingress load balancers operate at the
+application layer (L5-7). Incoming connections are terminated at the load balancer so it can inspect the individual HTTP /
+HTTPS requests. The requests are then forwarded via separate connections from the load balancer to the chosen service
+backing pods. As a result, network policy applied to the backing pods can restrict access to only allow connections from the load
+balancer, but cannot restrict access to specific original clients.
+
+### Why use Kubernetes Ingress?
+
+Given that Kubernetes [Services]({{site.baseurl}}/about/about-kubernetes-services) already provide a mechanism for load
+balancing access to services from outside of the cluster, why might you want to use Kubernetes Ingress?
+
+The mainline use case is if you have multiple HTTP / HTTPS services which you want to expose through a single external IP
+address, perhaps with each service having a different URL path, or perhaps as multiple different domains. This can be a
+lot simpler from a client configuration point of view compared to exposing each service outside of the cluster using
```

Comment:
reads a bit awkwardly....

Compared to exposing each service outside of the cluster using
Kubernetes Services, which gives each service it's own external IP address, this is lot simpler from a client configuration point of view.